.travis.yml

```diff
@@ -15,8 +15,10 @@ install:
       pip install numpy;
       pip install --pre torch -f https://download.pytorch.org/whl/nightly/cpu/torch_nightly.html;
       python setup.py build develop;
+      pip install pyro-ppl;
     else
       pip install torch==1.3.0+cpu -f https://download.pytorch.org/whl/torch_stable.html;
+      pip install pyro-ppl;
```

Comment:
Let's remove the changes to the travis.yml in favor of the ones merged in with the variational PR.